lib/Xmds/Soap5.php

```diff
@@ -476,6 +476,16 @@ public function RegisterDisplay(
             $value = $return->createTextNode($syncGroup->syncPublisherPort ?? 9590);
             $syncPublisherPortNode->appendChild($value);
             $displayElement->appendChild($syncPublisherPortNode);
+
+            $syncSwitchDelayNode = $return->createElement('syncSwitchDelay');
+            $value = $return->createTextNode($syncGroup->syncSwitchDelay ?? '');
```

Comment:
We should return the default values for these options, rather than empty. I think it would be handled correctly on the player as-is, but its clearer to always have them set.